library/core/tests/render/HeaderTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: rdown
+ * Date: 5/25/17
+ * Time: 02:02
+ */
```

Comment:
If keep this, it should get a standard license header.
Also need to ensure it doesn't get lost/remove in the future (maybe a comment as to what it does or will do)